src/language/providers/intellisense/elementItems.ts

```diff
@@ -49,17 +49,17 @@ export const elementCompletion = (definedVariables, nsPrefix) => {
       },
       {
         item: 'dfdl:assert',
-        snippetString: '<' + nsPrefix + 'assert $0',
+        snippetString: '<dfdl::assert $0',
```

Comment:
@rthomas320 Is this line, line 57 and line 58 supposed to `dfdl:` instead of `dfdl::`?